exporters/pelorus/utils.py

```diff
@@ -142,3 +145,36 @@ def format(self, record):
             return logging.Formatter.format(self, record)
         finally:
             self._style._fmt = prior_format
+
+
+def get_env_var(var_name: str, default_value: Optional[str] = None) -> str:
+    """
+    `get_env_var` modifies standard os.getenv behavior to allow using default python variable values
+    when:
+        1. PELORUS_DEFAULT_KEYWORD is set in SHELL env and the value from PELORUS_DEFAULT_KEYWORD
+           is used for other SHELL env value, e.g.
+           export PELORUS_DEFAULT_KEYWORD="custom_default"
+           export LOG_LEVEL="custom_default"
+
+           In which case LOG_LEVEL is set to DEFAULT_LOG_LEVEL
+
+        2. DEFAULT_VAR_KEYWORD keyword is present as the SHELL env variable value, e.g.
+           unset PELORUS_DEFAULT_KEYWORD
+           export LOG_LEVEL="default"
+
+           In which case LOG_LEVEL is set to DEFAULT_LOG_LEVEL
+
+    This is required for the config map to define fallback vars in a consistent way.
+    """
+    default_keyword = DEFAULT_VAR_KEYWORD
+    custom_default_keyword = os.getenv("PELORUS_DEFAULT_KEYWORD")
+    if custom_default_keyword and len(custom_default_keyword) > 0:
+        default_keyword = custom_default_keyword
```

Comment:
Since `None` and `""` are both falsy, this section can be simplified to `default_keyword = os.getenv("PELORUS_DEFAULT_KEYWORD") or DEFAULT_VAR_KEYWORD`

Reply:
changed.

Reply:
Yes I will change the config map PR later to include "default" when we expect default value.